rpc/common/admin/logging/moc_common_logging.proto

```diff
@@ -6,6 +6,14 @@ option go_package = "github.com/microsoft/moc/rpc/common/admin";
 package moc.common.admin;
 import "google/protobuf/wrappers.proto";
 
+enum VerbosityLevels {
```

Comment:
We are duplicating everything here. Can we keep just one file and use it both cloud and node agent?